tests/more-tests/subscript_individually_defined_stocks2/test_subscript_individually_defined_stocks2.mdl

```diff
@@ -0,0 +1,132 @@
+{UTF-8}
```

Comment:
Would this model be useful to  include in the test-models repo? Not sure if its something other people would find useful?

Reply:
This model is related to the dependencies dict and multiples dependencies. In fact, it is not run by integration test but used unit_test_pysd.py for testing these multiple dependencies management. I don't think that this will be useful for other projects, as it is just a simple copy  of test_subscript_individually_defined_stocks.mdl  with some small changes to test the dependencies.